src/test/scala/nl.knaw.dans.easy.file2bag/FilesXmlSpec.scala

```diff
@@ -30,7 +30,7 @@ class FilesXmlSpec extends AnyFlatSpec with Matchers {
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://purl.org/dc/terms/ http://dublincore.org/schemas/xmls/qdc/2008/02/11/dcterms.xsd http://easy.dans.knaw.nl/schemas/bag/metadata/files/ http://easy.dans.knaw.nl/schemas/bag/metadata/files/files.xsd">
         <file filepath="data/path/to/file.txt">
-            <dcterms:format>text/plain</dcterms:format>
+            <dc:format>text/plain</dc:format>
```

Comment:
shouldn't the namespace in r.28 also be changed to `dc`?

Reply:
The `dc` prefix is usually accompanied with a `xmlns:dc='http://purl.org/dc/elements/1.1/'`. Having said that, I also see a potential problem at line 61 and/or 92.

Reply:
these changes fall away after the last changes